src/c++/vdo/tests/FormatVDO_t3.c

```diff
@@ -49,16 +46,44 @@ static void minimumVDOTest(void)
   freeUserVDO(&vdo);
 }
 
+/**********************************************************************/
+static void minimumVDOTestInKernel(void)
+{
+  const TestParameters parameters = {
+    .indexMemory        = UDS_MEMORY_CONFIG_256MB,
+    .journalBlocks      = DEFAULT_VDO_RECOVERY_JOURNAL_SIZE,
```

Comment:
Not sure where to put this comment. Along with actually having a zeroing function, the format tests should maybe write junk in relevant blocks (UDS superblock, first journal bock, block map root blocks, etc.) so they can actually prove that the format is zeroing what it's supposed to be zeroing.

Reply:
I'll try to write another unit test to do this.